.dockerignore

```diff
@@ -0,0 +1,32 @@
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
+
+# Package Files
+*.jar
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+### Maven template
+target/
```

Comment:
Could this line exclude all the target directories of submodules?

Reply:
It seems that the `.dockerignore` is a little different from `.gitignore`. https://docs.docker.com/engine/reference/builder/#dockerignore-file

Maybe we need to use `**/target`, `**/*.iml`, etc.

Reply:
You're right. Will update.

Reply:
Updated 97fab7e